modules/programs/gpg.nix

```diff
@@ -30,6 +32,16 @@ in {
         <link xlink:href="https://gnupg.org/documentation/manpage.html"/>.
       '';
     };
+
+    keyfiles = mkOption {
```

Comment:
It would be great to have an option to also give the ownertrust of a key. This information is stored in the file `.gnupg/trustdb.gpg`. Unfortuantely, it is regularly written to by GnuPG even on operations which one would regard as read-only. Therefore, symlinking into the Nix store (as I'd suggest for the public keys, see below) would likely cause unexpected issues. It could be a middle ground to copy the `trustdb.gpg` file instead of symlinking it.